.github/workflows/build.yaml

```diff
@@ -8,6 +8,8 @@ on:
   workflow_dispatch:
   schedule:
```

Comment:
Let's remove the `schedule` for now? I took this workflow from a different project and didn't care much about it, but it might be the right time to remove it since we are cleaning up the project. Note that if we do that, we also need to remove some conditions from `jobs`.